### ISSUE TITLE
Connection error

### ISSUE THREAD

**Issue by AlexCroydon** [r3-dev/telegram-marketplace-miniapp#8]
SDK was unable to initialize. Probably, current application is being used not in Telegram Web Apps environment.
Unable to extract launch params. Errors: "Unable to parse field "tgWebAppVersion"", "Launch params are missing in local storage

**Comment by YSX7** [comment]
Are you trying open it in browser? This will not work, you need open in through telegram bot

**Comment by AlexCroydon** [comment]
I've set up the app using Docker, and when I try to add the link http://127.0.0.1:3000/market in the BotFather, Telegram doesn't allow URLs without HTTPS.

**Comment by AlexCroydon** [comment]
2024/05/12 00:17:38 Server started at http://0.0.0.0:3000
├─ REST API: http://0.0.0.0:3000/api/
└─ Admin UI: http://0.0.0.0:3000/_/

**Comment by AlexCroydon** [comment]
![screencapture-localhost-3040-2024-05-12-01_34_24](https://github.com/r3-dev/telegram-marketplace-miniapp/assets/65297976/1d8e3261-a064-478f-b1f7-c2f649e53495)

**Comment by YSX7** [comment]
Yeah, you need https. Another way to test project is to change frontend port to 3000 in frontend/package.json and backend/main.go, then go to Telegram test server and check @intgmarketbot and @intgstorebot
